viewer/config.txt

```diff
@@ -19,26 +19,27 @@ Center Frame         : true
 Frame Maximized      : false
 Save Frame State     : true
 
-Overlay Default Visibility:
+# Overlay Default Visibility
 Server Speed         : true
 Foul Overlay         : true
 Field Overlay        : true
 Number of Players    : true
 Player IDs           : true
 
-Networking Settings:
+# Networking Settings
 Auto-Connect         : true
 Auto-Connect Delay   : 1000
-Server Hosts         : localhost
-Server Port          : 3200
+Default Server Host  : localhost
+Default Server Port  : 3200
 Drawing Port         : 32769
+Server               : localhost,3200
```

Comment:
It looks strange to have host and port separated by a comma, normally a `:` is used for that.

Reply:
How does specifying multiple servers work now, do you now just put multiple `Server` lines like with `Team Color`? If so, we should probably have a comment explaining that as it's not intuitive / people won't discover this on their own.

Reply:
My idea was to have a different separator for the "value-value" pairs than for the "key-value" pairs. But I see, this may be confusing. I'd change it to a `:` for both the servers and the team colors (for consistency).

Yes, multiple servers can be specified by adding multiple `Server` lines. However, it is also possible to specify them by either editing them in `RVConfigure` (in the server list dialog) or by selecting the "Add to config file" check button in the "Connect to" dialog of RoboViz.
I'm not really sure where to document this properly. What would you suggest? (Or is this comment sufficient enough?)

Reply:
There could just be a comment above this line, like the one you added for possible look and feel values?